matlab_simulation/05-estimation/models/airplane_radar/airplane_radar_linearized_measurement_model.m

```diff
@@ -0,0 +1,3 @@
+function M = airplane_radar_linearized_measurement_model(mup, ~)
```

Comment:
Maybe comment in the equation that is shown in the notes so that people can follow along.